package.json

```diff
@@ -102,7 +103,10 @@
   "browserslist": "chrome 70, firefox 70, safari 11.1",
   "prettier": {
     "arrowParens": "avoid",
-    "singleQuote": true
+    "singleQuote": true,
+    "importOrder": ["^[./]"],
```

Comment:
This plugin considers everything not defined here a third party library.

With this regular explresion we are telling prettier to create one group for third parties, and another group for everything starting with `.` or `/`.

We could add more granular regular expressions here and the plugin would be able to create more groups if desired.